spec/factories/declarable.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :declarable do
+    trait :early_career_teacher_profile_declaration do
+      early_career_teacher_profile_declaration
+    end
+    trait :mentor_profile_declaration do
+      mentor_profile_declaration
+    end
+  end
+end
```

Comment:
Is this useful anymore?

Reply:
No. Probably not. Should be linking directly to the profiles.